pkg/services/sqlstore/migrations/accesscontrol/dashboard_permissions.go

```diff
@@ -41,13 +41,11 @@ var folderPermissionTranslation = map[models.PermissionType][]string{
 		dashboards.ActionDashboardsCreate,
 		dashboards.ActionFoldersRead,
 		dashboards.ActionFoldersWrite,
-		dashboards.ActionFoldersCreate,
```

Comment:
nit: I don't think it really matters in this case, but we shouldn't change existing migrations.